ts/lib/components/ButtonToolbar.svelte

```diff
@@ -39,11 +39,6 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     .button-toolbar {
         flex-wrap: var(--buttons-wrap);
         padding-left: 0.15rem;
-
-        :global(.button-group) {
-            /* TODO replace with gap once available */
-            margin-right: 0.15rem;
-            margin-bottom: 0.15rem;
-        }
+        gap: 1rem;
```

Comment:
That comment was from 2021, and `gap` is supposedly [well supported since April 2021](https://caniuse.com/flexbox-gap). Not sure which client was the blocker at the time, but hopefully it's fine now.
(The `margin` solution was broken.)

Reply:
Qt5 is Chromium 77, so this solution won't work with the Qt5 build unfortunately. The new spacing is also a bit too big I think - it used to be smaller.

Reply:
Pity, I'll revert to using `margin`.

In my opinion, there are many places where a bit more spacing wouldn't hurt, but since that's more or less a matter of taste, I tried to hold off from such changes.
However, in this case there wasn't any margin at all. Does it look different on a Mac?

Before:
![grafik](https://github.com/ankitects/anki/assets/48286681/e0a74557-cf4f-43e1-b72c-2ddf63cb5b0f)

Now:
![grafik](https://github.com/ankitects/anki/assets/48286681/b64a7de6-5f32-4357-9d51-c097bf7505af)

Reply:
I agree the latter screenshot looks better. I was looking at the editor when I made my previous comment - the buttons feel a bit widely-spaced now:
![2024-05-01T17:42:57,625111958+10:00](https://github.com/ankitects/anki/assets/180542/c1d10cd5-1cfd-4956-8612-e49377ef1707)

It was perhaps a bit too narrow before:
![2024-05-01T17:45:23,849219010+10:00](https://github.com/ankitects/anki/assets/180542/ea33cf04-beed-49bc-96e1-f3f4bbd44a9c)

So maybe somewhere in between might be nice :-)

Reply:
Whoops, that was unintended. I only meant to change the 3 pages mentioned above. Thanks for noticing! I'll push a follow-up.